Sources/NIOCore/EventLoop+SerialExecutor.swift

```diff
@@ -43,6 +43,18 @@ extension NIOSerialEventLoopExecutor {
     public var executor: any SerialExecutor {
         self
     }
+
+    @available(macOS 14.0, iOS 17.0, watchOS 10.0, tvOS 17.0, *)
+    @inlinable
+    public func isSameExclusiveExecutionContext(other: Self) -> Bool {
```

Comment:
This is not going to be called yet.

You must adjust `asUnownedSerialExecutor` to return `UnownedSerialExecutor` with the marker flag (there's a `complexEquality:` initializer.